src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -79,6 +80,7 @@ private Properties buildProperties() {
         props.setProperty("resource.loader", "class");
         props.setProperty("class.resource.loader.class", ClasspathResourceLoader.class.getCanonicalName());
         props.setProperty("runtime.log", new File(configuration.getReportDirectory(), "velocity.log").getPath());
+        props.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
```

Comment:
First one as I don't want to change data from the json file. Whatever you pass as doc_string it will be displayed as it is. This allows you to have html code in generated report. If you want to escape html into plain text maybe you should do it by yourself before report is started?

I have doubts for this so at this point all cases are fine as long as more users expect to have doc_string escaped let's leave version you prepared. In case of problem we will investigate and prepare new PR